commit 652e8b1e48a0acc521aa9ac1ff76f1743bbc03ef (refs/changes/16/2116/1)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-06-16T11:22:41-07:00 (2 years, 4 months ago)
    
    joyent/node-zkstream#11 Client#list() should return the stat as well

diff --git a/lib/client-fsm.js b/lib/client-fsm.js
index 14f9fbf..97d6457 100644
--- a/lib/client-fsm.js
+++ b/lib/client-fsm.js
@@ -582,13 +582,13 @@ ZKClientFSM.prototype.list = function (path, cb) {
 	mod_assert.string(path, 'path');
 	mod_assert.optionalFunc(cb, 'callback');
 	var req = this.request({
-		opcode: 'GET_CHILDREN',
+		opcode: 'GET_CHILDREN2',
 		path: path,
 		watch: false
 	});
 	if (cb) {
 		req.once('reply', function (pkt) {
-			cb(null, pkt.children);
+			cb(null, pkt.children, pkt.stat);
 		});
 		req.once('error', cb);
 	}
diff --git a/package.json b/package.json
index 35b9e86..ba20108 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "zkstream",
-  "version": "0.5.0",
+  "version": "0.6.0",
   "description": "",
   "main": "lib/index.js",
   "dependencies": {
